testctrl/kubernetes/images/driver/run.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Copyright 2015 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+SCENARIOS="{\"scenarios\":[$(echo $SCENARIO_JSON)]}"
+bins/opt/qps_json_driver --scenarios_json=$SCENARIOS
```

Comment:
Unfortunately, this json must be totally minified to work due to the way the shell handles arguments.

This is valid:

```json
{"name":"java_generic_..."}
```

This is not (one extra space after colon):
```json
{"name": "java_generic_..."}
```